load_testing/configs/1_200_ramp_config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "locustfile_list": ["call_aaq.py"],
+  "users_list": [10, 20, 30, 50, 100, 150, 200],
+  "spawn_rate_list": [1, 2, 3, 5, 5, 5, 10],
```

Comment:
Why ramped btw? Is it not better to just spawn the total and let them loose? That way we can interpret the average response times etc of the report (as per what's written in the README)

Reply:
@amiraliemami Good suggestion - folded in 🙏